### DISCUSSION TITLE
Sample Workflow for face Swap?

### DISCUSSION THREAD

**Discussion by Piscabo** [melMass/comfy_mtb#29] [General]
Is there any info to help a rookie create a face-swap workflow? I get this error...
[comfy_mtb.nodes.faceswap] | INFO -> Running insightface swap (batch size: 1)
[comfy_mtb.nodes.faceswap] | WARNING -> No target face found for 1
[comfy_mtb] | WARNING -> Weight value has no effect for now. (value: 1.0)

**Comment by melMass** [comment]
There will be examples in the next release, you can give it a go, but I'm not sure they would work with the current version!

https://github.com/melMass/comfy_mtb/tree/dev/next-release/examples
![georges lucas faceswap example](https://user-images.githubusercontent.com/7041726/254113676-272af7d6-f01c-478e-a82f-926e772d7209.png)

**Reply by Piscabo** [reply]
There is no embedded workflow.  And 01-faceswap.json lods nothing? I'm just going to start over to see if there is another problem first. Otherwise, I will report back... Thanks for your help!!
[comfy_mtb] | WARNING -> cv2.ximgproc.guidedFilter not found, use opencv-contrib-python
[comfy_mtb] | ERROR -> Some nodes failed to load:
        Failed to import faceswap because ModuleNotFoundError: No module named 'onnx.onnx_cpp2py_export.checker'; 'onnx.onnx_cpp2py_export' is not a package

Check that you properly installed the dependencies.
If you think this is a bug, please report it on the github page (https://github.com/melMass/comfy_mtb/issues)
